packages/samples/test/output/visibility/@typespec/openapi3/openapi.yaml

```diff
@@ -103,6 +103,8 @@ components:
         id:
           type: string
           readOnly: true
+        secret:
+          type: string
```

Comment:
Since `OptionalPerson` is unreferenced, this behavior is correct, and an example of the breaking behavior.